src/routes/resources/what-is-rei/+page.svelte

```diff
@@ -1,5 +1,9 @@
 <svelte:head>
 	<title>What is Rei? | Kendo Club at the University of Michigan</title>
+	<meta
+		name="description"
+		content="Read an enlightening perspective about rei, an important part of kendo philosophy ."
```

Comment:
Do we want to include the author? I know some of google’s examples included an author.

Reply:
Resolved in 6b51d18dfdea95912165a5d3b7043a537c756c83